### DISCUSSION TITLE
Install AutoMapper in Xamarin Forms 5

### DISCUSSION THREAD

**Discussion by jbdioli** [LuckyPennySoftware/AutoMapper#4012] [NA]
Hello

I'm trying to install AutoMapper 11.0.1 or another old version, in Xamarin Forms 5, and I get always the same error message:

Package restore failed. Rolling back package changes for 'AutoMapper'.
Cycle detected.
AutoMapper -> AutoMapper (>=10.1.1)

Thanks for your help

**Comment by lbargaoanu** [comment]
https://github.com/AutoMapper/AutoMapper/issues/2566

**Reply by jbdioli** [reply]
Thanks a lot! I've recreated the project with another name and it works.
